features/keychain/index.js

```diff
@@ -4,9 +4,7 @@ import keychainDefinition from './module/keychain'
 const keychain = ({ cachePublicKeys }) => {
   return {
     id: 'keychain',
-    definitions: cachePublicKeys
-      ? [{ definition: memoizedKeychainDefinition, storage: { namespace: 'keychain' } }]
-      : [{ definition: keychainDefinition }],
```

Comment:
cachine is based on single seed atm, and we're not using it anywhere, so i'm tmp killing it